### PR TITLE
CSV output columns consistent with provided language [former PR32]

### PR DESCRIPTION
**Pull request by fonsecadeline**

### CODE REVIEW

**Review by braktar — APPROVED** [review]

**Review by senhalil — APPROVED** [review]
@fonsecadeline I feel like the renaming done in PR #174  might create a problem for this PR, am I right? If there is a risk, which one needs to be merged first, can you add a task so that we don't merge them in wrong order.

**Comment by fonsecadeline** [comment]
@senhalil @braktar I added one commit because I realized that _use_deprecated_csv_headers_ was not working as expected from front page. In fact we used to modify I18n value when calling vrp.create. But when calling GET with a different language it did not "remember" we had asked for _use_deprecated_csv_headers_.

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
 - Bump OR-Tools v7.8 [#107](https://github.com/Mapotempo/optimizer-api/pull/107)
 - VROOM were previously always called synchronously, it is now reserved to a set of effective `router_mode` (:car, :truck_medium) within a limit of points (<200). [#107](https://github.com/Mapotempo/optimizer-api/pull/107)
 - Heuristic selection (`first_solution_strategy='self_selection'`) takes into account the supplied initial routes (`routes`) and the best solution is used as the initial route [#159](https://github.com/Mapotempo/optimizer-api/pull/159)
+- CSV headers adapts to the language provided to facilitate import in Mapotempo-Web [#32](https://github.com/Mapotempo/optimizer-api/pull/32)
```
Comment (fonsecadeline) [inline CHANGELOG.md:22]:
```suggestion
- CSV headers adapts to the language provided through HTTP_ACCEPT_LANGUAGE header to facilitate import in Mapotempo-Web [#32](https://github.com/Mapotempo/optimizer-api/pull/32)
```

**Comment by fonsecadeline** [comment]
I had to force push after rebasing with dev and fixing some errors. Non reviewed changes are in https://github.com/Mapotempo/optimizer-api/pull/196/commits/77fa93d2b06e78ac4a87fea8bce2824c589c1fb7

**Comment by fonsecadeline** [comment]
- [ ] we need to tell customers before merging this PR

**Review by senhalil — APPROVED** [review]

File: models/configuration.rb [hunk 2]
```diff
@@ -79,11 +79,13 @@ class Restitution < Base
     field :geometry, default: []
     field :intermediate_solutions, default: true
     field :csv, default: false
+    field :use_deprecated_csv_headers, default: false
     field :allow_empty_result, default: false
   end
 
   class Schedule < Base
     field :range_indices, default: nil # extends schedule_range_date
+    field :range_date, default: nil # should only be used to return consistent date in result. TODO: move to result structure
```
Comment (braktar) [inline models/configuration.rb:88]: To avoid any misuse of this parameter, could we only store the starting date ?

Comment (fonsecadeline) [reply models/configuration.rb:88]: Good idea !

**Review by braktar — APPROVED** [review]

**Pull request merged**